README.md

```diff
@@ -832,6 +832,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1600+ peo
   * [Grafana Cloud](https://grafana.com/products/cloud/) - Grafana Cloud is a composable observability platform that integrates metrics and logs with Grafana. Free: 3 users, ten dashboards, 100 alerts, metrics storage in Prometheus and Graphite (10,000 series, 14 days retention), logs storage in Loki (50 GB of logs, 14 days retention)
   * [healthchecks.io](https://healthchecks.io) — Monitor your cron jobs and background tasks. Free for up to 20 checks.
   * [Hydrozen.io](https://hydrozen.io) — Uptime monitoring & Status pages, Free plan: 10 Uptime monitor, 5 heartbeat monitors, 1 Domain monitor and 1 Statuspage free.
+  * [incidenthub.cloud](https://incidenthub.cloud/) — Status page monitoring aggregator, 20 monitors and 2 notification channels are free
```

Comment:
unfortunately pricing is not clear so I can't verify this.

<img width="232" alt="Screenshot 2024-08-30 at 10 59 29" src="https://github.com/user-attachments/assets/2f8897c4-d4cd-4f4a-b842-50e7c8e3f929">

Reply:
Ok, let me reopen the PR once I have the pricing (free + paid) grid in place (still working on it). Thanks.